src/nepiada/env/nepiada.py

```diff
@@ -167,11 +168,9 @@ def step(self, actions):
         observations = self.observations
```

Comment:
We should probably update this to now return that we calculate right?

Reply:
Yes, good catch. It's stored in a different variable though, check my top most comment for clarity